enhancements/olm/operator-bundle.md

```diff
@@ -105,17 +105,54 @@ An operator author can also specify the version of the format used inside the bu
 operators.operatorframework.io.bundle.mediatype: "registry+v1"
 ```
 
+###### Graph Metadata
+An additional point of concern is the fact that the current [operator-registry manifest format](https://github.com/operator-framework/operator-registry#manifest-format) also has a multi-version aggregation file `package.yaml` that describes data about the entire set of releases. Given that we are splitting up these manifests into separate objects, we now need a way to denormalize this data so that it applies only to an individual release. In order to do this, we will create a new `registry.yaml` file. Previously the `package.yaml` file was defined like this:
+
+*package.yaml*
+```yaml
+packageName: etcd
+channels:
+- name: singlenamespace-alpha
+  currentCSV: etcdoperator.v0.9.4
+- name: clusterwide-alpha
+  currentCSV: etcdoperator.v0.9.4-clusterwide
+defaultChannel: singlenamespace-alpha
+```
+
+The data here defined a package name to attach to all versions, a default channel that defines what channel a user should subscribe to if they are not opinionated about how they want to subscribe to updates, and information about the head of every channel. Now that this data needs to be defined *per release*, we will make the following changes:
+
+1. Retain the Package Name
+2. Remove the concept of the Default Channel from individual packages. Instead we will define a heuristic to determine what the default channel is for now when installing the operator.
```

Comment:
What does this heuristic look like?

Reply:
There was a big discussion on the original PR - I think we're punting for now.